src/absil/ilreflect.fs

```diff
@@ -1235,9 +1271,12 @@ let emitCode cenv modB emEnv (ilG:ILGenerator) code =
 let emitLocal cenv emEnv (ilG : ILGenerator) (local: ILLocal) =
     let ty = convType cenv emEnv  local.Type
     let locBuilder = ilG.DeclareLocalAndLog(ty, local.IsPinned)
+#if NO_PDB_WRITER
```

Comment:
Because there is a ton of code that needs rewriting between the #if and #endif and the codebase still compiles the full desktop version of the compiler and protocompiler.

These change follow the existing F# pattern of 
   FX_SOMEFEATURE or FX_NO_SOMEFEATURE

to indicate a large block of code that is optional between compile targets.  And yes that should be: FX_NO_PDB_WRITER and FX_NO_PDB_READER.

Although they were pre-existing defines,:
https://github.com/Microsoft/visualfsharp/search?utf8=%E2%9C%93&q=NO_PDB_READER

I think I will still change them.